### PR TITLE
Update styles.css

### PR DESCRIPTION
**Pull request by satcasm**
## What is the change?
Made the margin better under the navbar.

## Related issue?
issue: #232 

## Screenshots or Video:
image

### CODE REVIEW

**Comment by satcasm** [comment]
Hey @Pranav016 , can you tell me why is it  getting failed here ? I can't comprehend the error message.

**Comment by Pranav016** [comment]
> Hey @Pranav016 , can you tell me why is it getting failed here ? I can't comprehend the error message.

Don't worry about it..

**Comment by satcasm** [comment]
Also resolved issue: #234

File: public/css/styles.css [hunk 1]
```diff
@@ -389,6 +389,9 @@ body {
   transform: rotate(360deg);
   text-indent: 0.1em;
 }
+.normal-heading-font {
+  margin-top: 25px;
```
Comment (Pranav016) [inline public/css/styles.css:393]: Use relative units of measurement (eg. rem, em ,vh, vw) to keep website responsive.

**Review by Pranav016 — CHANGES_REQUESTED** [review]
* Don't commit to master branch. Work on a new branch
* Make separate PRs for separate issues.
* Set base branch to develop when making the PR
* Read contribution guidelines

**Comment by Pranav016** [comment]
Closing PR, make a new one from new branch.
Reopen PR if required

**Pull request closed without merging**